### ISSUE TITLE
Windows compilation guide: "error: target not found: mysys2-runtime"

### ISSUE THREAD

**Issue by markwkidd** [libretro/docs#52]
I'm following the Core compilation guide for Windows step-by-step: https://github.com/libretro/docs/blob/master/docs/compilation/windows.md

The second `pacman` command fails with the error `error: target not found: mysys2-runtime`

Here is the text of the command: `pacman --needed --noconfirm -S bash pacman pacman-mirrors msys2-runtime`

**Comment by markwkidd** [comment]
It seems that if one closes the mysys shell after the first `pacman` command, then restarts the shell, the second command does not produce an error. I'll keep moving forward with the process and submit a PR if that is the only change needed to what's written.

**Comment by fr500** [comment]
sometimes the packages change over time so it may need updating yeah
Some markup is not supported by git it seems.

The render should be like this 
https://bot.libretro.com/docs/compilation/windows/

**Comment by markwkidd** [comment]
I believe this was an early sign of motherboard failure. Within a few more weeks I began getting more serious, but intermittent errors.

More recently I followed this guide on another Windows 10 PC and had an experience much closer to the one described. I'm closing this issue for now but as I get my new laptop setup I'll be looking for anything that does seem like a legitimate need for an update. Thank you for your help so far.
